content/2017.04.getting-started-with-elm.rst

```diff
@@ -0,0 +1,487 @@
+Getting started with Elm
+########################
+
+:lang: en
+:date: 2017-04-13
+:url: getting-started-with-elm
+:summary: What is Elm? how does it work? How can I try it really fast?
+
+
+Introduction
+============
+
+The past two weeks we have been traveling and meeting a bunch of
+developers.
+
+It was a chance for us to tell them about Elm through talks at
+conferences and to present a short one-hour workshop.
+
+You can find the slides of the two talks here:
+
+- `a short introduction to Elm by Niko <http://slides.com/n1k0/elm>`_
+- `Talking JSON with Elm by Natim <http://natim.github.io/django-slides/elm-json/>`_
+
+However, this article is a summary of the take aways of this workshop
+for those who would like to know a bit more about Elm.
+
+
+What is Elm and why should I be interested?
+===========================================
+
+Elm is a toolkit that is focused on making sure your app state and the
+HTML that reflects it are always in sync.
+
+If you have been doing any frontend development during the last couple
+of years, you might have seen that the JavaScript ecosystem is a bit
+wild. Keeping up-to-date with the growing number of competing
+frameworks can take a lot of energy, not to mention trying your app on
+all the available browser versions to make sure it works well
+everywhere. Training a team to build an app with React.js,
+Angular2.js, Vue.js or Ember.js as well as looking for the right
+boilerplate to get started with your project are good examples of how
+hight the frustration that the constant reinvention of the ecosystem
+causes can be.
+
+On the other hand, writing tests for the front-end can be tedious and
+supporting Flow or TypeScript can prove to be yet another layer to
```

Comment:
I'd also add that these typecheckers are actually useful. Eg. *supporting Flow or TypeScript, while tremendously useful, is yet another layer to setup, configure and maintain in your stack.*